hack/update-generated-deep-copies.sh

```diff
@@ -55,6 +55,12 @@ VERSIONS="api v1beta3 v1"
 for ver in $VERSIONS; do
 	rm -f `result_file_name ${ver}`
 done
-for ver in $VERSIONS; do 
-	generate_version "${ver}"
+KUBE_API_VERSIONS=""
+for ver in $VERSIONS; do
+  # Ensure that the version being processed is registered by setting
+  # KUBE_API_VERSIONS.
+  if [ "${ver}" != "api" ]; then
+    KUBE_API_VERSIONS="${ver}"
+  fi
+	KUBE_API_VERSIONS="${KUBE_API_VERSIONS}" generate_version "${ver}"
```

Comment:
Fix indentation. Also, I'd recommend making a local variable instead of reusing KUBE_API_VERSIONS like this.